sdk/python/responsible-ai/text/responsibleaidashboard-text-classification-blbooksgenre.ipynb

```diff
@@ -447,7 +451,8 @@
     "                   max_retries=max_retries,\n",
     "                   retry_delay=retry_delay)\n",
     "    model = AutoModelForSequenceClassification.from_pretrained(\n",
-    "        BLBOOKSGENRE_MODEL_NAME, num_labels=NUM_LABELS)\n",
+    "        BLBOOKSGENRE_MODEL_NAME, num_labels=NUM_LABELS,\n",
+    "        trust_remote_code=True)\n",
```

Comment:
you only need this for the dataset AFAIK and not for model

Reply:
I removed the change for model. Let's see if the notebook can pass the test